dandi/metadata.py

```diff
@@ -455,11 +455,22 @@ def extract_sex(metadata: dict) -> Optional[models.SexType]:
 def extract_strain(metadata: dict) -> Optional[models.StrainType]:
     value = metadata.get("strain", None)
     if value is not None and value != "":
+        # Don't assign cell lines to strain
+        if value.lower().startswith("cellline:"):
+            return None
         return models.StrainType(name=value)
     else:
         return None
 
 
+def extract_cellLine(metadata: dict) -> Optional[str]:
+    value = metadata.get("strain", None)
+    if value is not None and value != "" and value.lower().startswith("cellline:"):
+        return value.split("cellline: ")[1]
```

Comment:
To be inline with above check (no trailing space) and to ensure absent additional splitting and no leading/trailing whitespaces

```suggestion
        return value.split("cellline:", 1)[1].strip()
```